src/pybind/mgr/cephadm/services/nvmeof.py

```diff
@@ -30,7 +30,7 @@ def prepare_create(self, daemon_spec: CephadmDaemonDeploySpec) -> CephadmDaemonD
         host_ip = self.mgr.inventory.get_addr(daemon_spec.host)
 
         keyring = self.get_keyring_with_caps(self.get_auth_entity(nvmeof_gw_id),
-                                             ['mon', 'profile rbd',
```

Comment:
should create a new profile for a more restrictive set of capabilities, e.g. just the monitor commands necessary